packages/grz-common/src/grz_common/transfer.py

```diff
@@ -31,8 +31,20 @@ def init_s3_client(s3_options: S3Options) -> S3Client:
     """Create a boto3 Client from a grz-cli configuration."""
     # configure proxies if proxy_url is defined
     proxy_url = s3_options.proxy_url
+    proxy_config_options = s3_options.proxy_config
+    proxies_config = None
+    if proxy_config_options is not None:
+        proxy_ca_bundle = proxy_config_options.proxy_ca_bundle
+        proxy_client_cert = proxy_config_options.proxy_client_cert
+        proxy_use_forwarding_for_https = proxy_config_options.proxy_use_forwarding_for_https
+        proxies_config={"proxy_use_forwarding_for_https": proxy_use_forwarding_for_https}
+        if proxy_ca_bundle is not None: 
+            proxies_config["proxy_ca_bundle"] = str(proxy_ca_bundle)
+        if proxy_client_cert is not None: 
+            proxies_config["proxy_client_cert"] = str(proxy_client_cert)
```

Comment:
Since `ProxyOptions` has matching field names, I think it may make sense to use it directly, e.g.:
```py
    proxies_config = (
        s3_options.proxy_config.model_dump(exclude_none=True)
        if s3_options.proxy_config
        else None
    )
```